package.json

```diff
@@ -39,12 +39,13 @@
     "dist"
   ],
   "peerDependencies": {
-    "vue": "^2.7.16"
+    "vue": "3"
```

Comment:
`"@vue/compiler-sfc": "^3.4.23"`, so here vue should be also `^3.4.23`, not `3`

Reply:
Or lower, because it will also work fine on a lower version

Reply:
`3.4.23` would be fine